### ISSUE TITLE
Settings panel opening up in odd place

### ISSUE THREAD

**Issue by mrkeysh** [atom/settings-view#1092]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Settings panel opening up in random/odd places

### Steps to Reproduce

1. Open up Settings (Ctrl+, or File ---> Settings)

**Expected behavior:** [What you expect to happen]

Settings panel to open up in the main editor area with a tab

**Actual behavior:** [What actually happens]

settings panel seems to open up in the side panel (possibly the navigation panel)

**Reproduces how often:** [What percentage of the time does it reproduce?]

happens 1x, but then unable to "undo" this without a clean install...has happened about 4x in the past 3 months

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.
3 different computers, all Windows 10 64 bit
Most recently Atom 1.32.2, possibly as early as 1.2X? (didn't note when it first happened about 3 months ago)

### Additional Information

Normally the settings pane will open up in the main editor area.  Either by using Ctrl+, or File --> Settings (or any other way to get to that pane).
However, sometimes (and once it happens once it won't stop), this will open in what appears to be the "navigation pane".  I don't think its truly the navigation pane though, but another pane on top of the navigation pane.  
The only way to get out of this "settings" page is to then right-click on the area and select "close pane" (or Ctrl-W).
You can't get to the tab at the top, you can't move the pane or tab somewhere else.  

It almost seems as if this is the result of a bad package installed, but I don't know what package.  I have tried to uninstall packages and/or disable them to get it back to "normal".  But to no avail. 

Currently this is on the latest version of Atom (1.32.2) for windows 64 on Windows 10.  This has happened on 3 different Windows machines that I have used.  This has also happened on several versions of Atom, since about 1.2x? (I didn't really take note at the time, just did a Google search with no results and did a fresh reinstall).

Its hard to reproduce as I can't "undo" it without a clean reinstall.  I don't do anything out of the ordinary or different, its just one time seemingly random time I am looking to do something in my settings and I will hit Ctrl+, and then it pops up over the navigation pane.

Regular window
![image](https://user-images.githubusercontent.com/2074768/48561345-bea8da00-e8bd-11e8-83db-074ad8a7a8f7.png)

Settings page
![image](https://user-images.githubusercontent.com/2074768/48562326-f6b11c80-e8bf-11e8-930b-0f9b06ae2d15.png)

Trying to close the settings page
![image](https://user-images.githubusercontent.com/2074768/48562261-cff2e600-e8bf-11e8-8b4c-d0ab9fb6b937.png)

**Comment by rsese** [comment]
Thanks for opening a new issue in this repository :+1:

> The only way to get out of this "settings" page is to then right-click on the area and select "close pane" (or Ctrl-W).
> You can't get to the tab at the top, you can't move the pane or tab somewhere else.

The screenshots aren't in safe mode so just to clarify, if you run in safe mode can you drag and drop the settings view out of the right side bar?

**Comment by mrkeysh** [comment]
To clarify, the screenshots are not in safe mode.  I wanted to take the screenshots to show/illustrate where the panel was opening (on top of the projects pane).
I also currently don't have this issue as I did a fresh install.  However, when open in Safe Mode I was not able to drag the Settings view anywhere (nothing to grab onto to drag).
After some discussion it turns out some of this might be due the Theme I was using, which was hiding the "Projects" heading tab, the part of the tab you would normally grab onto to drag and drop anywhere.  And once the "settings" view was on that pane, the tab part that said "settings" would be hidden and not able to be dragged anywhere.
However, this still doesn't solve the problem of what cause the issue in the first place...which could also be some abberant thing from the UI theme.
I think the theme at the time I was using was City-lights.

**Comment by rsese** [comment]
> After some discussion it turns out some of this might be due the Theme I was using, which was hiding the "Projects" heading tab, the part of the tab you would normally grab onto to drag and drop anywhere. And once the "settings" view was on that pane, the tab part that said "settings" would be hidden and not able to be dragged anywhere.
...
> I think the theme at the time I was using was City-lights.

Ahh gotcha, that makes sense.  So once you switched to a default/some other theme, you were able to drag the settings over again.

> However, this still doesn't solve the problem of what cause the issue in the first place...which could also be some abberant thing from the UI theme.

That's true but we would need specific steps to reproduce the case where the settings view opens in a different location (or in the same pane as the tree view in particular).  It sounds like repro steps are unknown right now?

**Comment by mrkeysh** [comment]
> That's true but we would need specific steps to reproduce the case where the settings view opens in a different location (or in the same pane as the tree view in particular). It sounds like repro steps are unknown right now?

Correct.  This might also might not be the best place for this as it now seems more and more like it could be the theme itself causing something by accident, possibly through it hiding the "heading" of the "projects" pane.  
I completely understand that you (or anyone) needs the steps done in order to reproduce the error, which is why I am stating that and just making it clear.  Not trying to be clueless or helpless here, but I do completely understand and hoping that even without the steps to repro someone might go "OH...YEAH...THAT...." , but I know that is not normal.  
So I am going to try to switch back and actively watch for the steps done next week to see what was done to cause that issue.  
Sorry about that.  
Thank you again for the help
